.storybook/theme.ts

```diff
@@ -24,15 +28,17 @@ const Theme = create({
   barSelectedColor: '#1EA7FD',
 
   // Typography
-  fontBase: `"Nunito Sans", -apple-system, ".SFNSText-Regular", "San Francisco", BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif`,
+  fontBase:
+    primaryTheme.FONTS.baseFont +
```

Comment:
I added our fonts here so that the `Docs` page uses the respective theme font when toggling instead of the Storybook default. I explored making other changes but I couldn't really get most to show up, I think our `<Global />` resets override most of these theme configuration options.